docker/owlbot/python/cloudbuild.yaml

```
@@ -0,0 +1,10 @@
+steps:
+    - name: 'gcr.io/cloud-builders/docker'
+      args: [ 'build',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-python:$SHORT_SHA',
+        '-t', 'gcr.io/repo-automation-bots/owlbot-python:latest',
```

You might want to replace the hardcoded project with `$PROJECT_ID` which is a built in substitution so folks can run in their own projects without needing permissions.
```suggestion
        '-t', 'gcr.io/$PROJECT_ID/owlbot-python:$SHORT_SHA',
        '-t', 'gcr.io/$PROJECT_ID/owlbot-python:latest',
```

---

Same for the other instances of `repo-automation-bots` in this cloudbuild.yaml

---

Done.